### PR TITLE
[Rust] Implemented new APIs for Memory and Config.

### PR DESCRIPTION
**Pull request by apepkuss**
In this PR, new APIs are introduced into `Memory` and `Config`. In addition, the `bindings-rust` workflow is refactored.

### CODE REVIEW

File: .github/workflows/bindings-rust.yml [hunk 1]
```diff
@@ -25,11 +25,11 @@ on:
 
 jobs:
   build_ubuntu:
-    name: Ubuntu 20.04
+    name: Ubuntu-20.04 && Ubuntu-22.04
```
Comment (hydai) [inline .github/workflows/bindings-rust.yml:28]: I think you can use `Ubuntu` only.

**Review by hydai — CHANGES_REQUESTED** [review]

File: .github/workflows/bindings-rust.yml (hunk 1 above)
Comment (apepkuss) [reply .github/workflows/bindings-rust.yml:28]: Accepted. Will refactor it. Thanks!

**Comment by apepkuss** [comment]
@hydai The review issue has been fixed. Please help review the revision. Thanks!

**Review by hydai — APPROVED** [review]

**Review by hydai — APPROVED** [review]

**Comment by apepkuss** [comment]
@hydai Thanks for the review!

**Pull request merged**